src/Php/TypeAnalyzer.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Php;
+
+use Nette\Utils\Strings;
+
+final class TypeAnalyzer
+{
+    public function isPhpReservedType(string $type): bool
+    {
+        return in_array($type, ['string', 'bool', 'mixed', 'object', 'iterable', 'array'], true);
```

Comment:
Missing types:
- `float`
- `int`

And should we care about `boolean`, `double`? The variants?

Reply:
I'll add them, thanks